src/User.js

```diff
@@ -3,6 +3,21 @@ class User {
   this.id = id;
   this.name = name;
   }
+
+  userLogin(username, password, users) {
```

Comment:
be careful with the .()find, it doesn't return anything else other than the object or undefined. You are testing the right return, but your method is asking differently